public/js/views/window-view.js

```diff
@@ -24,7 +24,7 @@ var WindowView = Backbone.View.extend({
 
     evt.preventDefault();
 
-    require( '../router' ).goTo( targetUrl );
+    require( '../router' ).nav( targetUrl );
```

Comment:
Note to self: Stateman appears to have an "autolink" feature of some kind, I should investigate how that works.